src/adapters/abstractAdapter.js

```diff
@@ -25,12 +25,34 @@ class Adapter {
 
```

Comment:
can you rename `abstractAdapter.js` to `abstract-adapter.js` case sensitive file names are a pain in the ass on some OS.

Reply:
Will do.